package.json

```diff
@@ -15,13 +15,13 @@
   "devDependencies": {
     "@babel/preset-env": "^7.21.5",
     "babel-eslint": "^10.1.0",
-    "css-loader": "^6.7.3",
+    "css-loader": "^6.7.4",
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.27.5",
     "hint": "^7.1.8",
     "html-webpack-plugin": "^5.5.1",
-    "style-loader": "^3.3.2",
+    "style-loader": "^3.3.3",
     "stylelint": "^13.13.1",
```

Comment:
- [ ] [Optional] [Highly recommended] Please add `webpack-dev-server` to the devDependencies and also add `webpack`, `webpack-cli`, and `webpack-dev-server` to the regular dependencies by installing them like that:
```terminal
npm i webpack webpack-cli webpack-dev-server --save-dev 
```

Reply:
Thanks for the feedback! I will try to improve that.